src/app/sessions/services/sessions-creation.service.ts

```diff
@@ -33,7 +33,6 @@ export class SessionsCreationService {
 
         this.store.dispatch(new SessionCreationActions.InitializeTransaction(transaction));
         this.store.dispatch(new SessionActions.Update(session));
-        this.store.dispatch(new SessionActions.UpsertOne({...session, duration: moment.duration(session.duration * 1000)}));
```

Comment:
why removal?

Reply:
The session upsert should happen once the session is commited. At this point it is not. That line was a mistake from my previous story :(